roles/mediawiki/tasks/update.yml

```diff
@@ -48,8 +53,8 @@
       - name: MEDIAWIKI_ADMIN_PASS
         value: "{{ mediawiki_admin_pass }}"
       - name: MEDIAWIKI_SITE_SERVER
-        value: '{{ route.route.spec.host }}'
-      image: docker.io/ansibleplaybookbundle/mediawiki123:latest
+        value: '{{ endpoint.endpoints.subsets[0].addresses[0].ip }}'
+      image: docker.io/jmontleon/mediawiki123:update
```

Comment:
This doesn't look like a change you want to make.

Reply:
Shouldn't we be using the `mediawiki` image now instead of `mediawiki123`?

Reply:
My rebase scooped this up. It should be docker.io/ansibleplaybookbundle/mediawiki123:latest

Reply:
What I meant was the [`mediawiki123` image](https://hub.docker.com/r/ansibleplaybookbundle/mediawiki123/) isn't being updated anymore, you should probably use the [`mediawiki` image](https://hub.docker.com/r/ansibleplaybookbundle/mediawiki-apb/)